aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -281,13 +283,54 @@ class CAFFE2_API OperatorHandle final {
 
 namespace detail {
 template<class... Args> inline void unused_arg_(const Args&...) {}
+
+template <typename T, std::enable_if_t<c10::impl::not_ok_to_box<T>::value>* = nullptr>
+inline void push_ivalue_copy(std::vector<c10::IValue>& stack, const T& v) {}
+template <typename T, std::enable_if_t<!c10::impl::not_ok_to_box<T>::value>* = nullptr>
+inline void push_ivalue_copy(std::vector<c10::IValue>& stack, const T& v) {
+  torch::jit::push(stack, v);
+}
+
+template<typename Item>
+void convert_to_ivalue_vector(std::vector<c10::IValue>& stack, const Item& item) {
+  push_ivalue_copy(stack, item);
+}
+template<typename Item, typename... Rest>
+void convert_to_ivalue_vector(std::vector<c10::IValue>& stack, const Item& item, Rest... other_items) {
```

Comment:
Can we move this logic to the boxing.h file, maybe call it `boxArgumentsIntoStack()` and then also call it from `boxAndCallBoxedFunc` to reuse the code?